### DISCUSSION TITLE
How to save pytorch model trailed with Optuna ?

### DISCUSSION THREAD

**Discussion by averma111** [optuna/optuna#4695] [Q&A]
### Don't use GitHub Issues to ask support questions.

I am trying to tuen pytorch regression model with Optuna and able to get best results. Now I want to save the best trained model and use to predict test data, let me know how can we do it. 
here is the code =>https://www.kaggle.com/code/averma111/pytorch-ps3e15-optuna/edit

**Comment by nzw0301** [comment]
Did you try to save models and then load it after optimisation like https://optuna.readthedocs.io/en/stable/faq.html#how-to-save-machine-learning-models-trained-in-objective-functions?
